packages/dashmate/docker-compose.platform.yml

```diff
@@ -46,6 +46,7 @@ services:
     ports:
       - ${PLATFORM_DRIVE_TENDERDASH_P2P_PORT:?err}:${PLATFORM_DRIVE_TENDERDASH_P2P_PORT:?err} # P2P
       - 127.0.0.1:${PLATFORM_DRIVE_TENDERDASH_RPC_PORT:?err}:${PLATFORM_DRIVE_TENDERDASH_RPC_PORT:?err} # RPC
+      - 127.0.0.1:${PLATFORM_DRIVE_TENDERDASH_PPROF_PORT:?err}:${PLATFORM_DRIVE_TENDERDASH_PPROF_PORT:?err} # pprof
```

Comment:
It will work for regular nodes, but for local mode you will need also set different pprof ports for each local node or docker compose will error you about port is already bound.

Check this code sample: 

https://github.com/dashpay/platform/blob/da8548a47b9b6f1fb31b363d8ea7015406d40d46/packages/dashmate/src/listr/tasks/setup/setupLocalPresetTaskFactory.js#L261

Reply:
It might be needed to also reflect this in the migrations as well

https://github.com/dashpay/platform/blob/da8548a47b9b6f1fb31b363d8ea7015406d40d46/packages/dashmate/configs/migrations.js#L517

Reply:
Excellent point, I've added the port offset in the local setup command. I'm also not sure if migrations are needed here, because this port did not exist in config before, and now the offsets will be present with the correct values when this PR is merged, right?

Reply:
That is true for new local groups, but what if user already had a local group that did not have pprof option, and he downloaded new version, where it is required. Migration will not update current local configs and user will not be able to proceed with any commands